deploy/kubernetes/base/controller.yaml

```diff
@@ -232,6 +235,7 @@ spec:
             - --kube-api-qps=20
             - --kube-api-burst=100
             - --workers=100
+            - --retry-interval-start=4s
```

Comment:
Do we still want to increase this value, given that the resizer sidecar will soon retry any InvalidArgument RPC errors at retry-interval-max? That will already prevent misconfigured K8s resources from spamming EC2 backend, without making other types of retries take longer.  

What types of errors other than InvalidArgument would we still want our customers to have a delayed retry for?

Reply:
Resolving because of the EV modification cooldown being a large source of non-InvalidArgument RPC errors.